src/clusterfuzz/_internal/bot/fuzzers/libfuzzer.py

```diff
@@ -740,7 +740,10 @@ def _restart_qemu(self):
       with open(QemuProcess.LOG_PATH) as f:
         # Strip non-printable characters at beginning of qemu log
         qemu_log = ''.join(c for c in f.read() if c in string.printable)
-        logs.log_warn(qemu_log)
+        # Only report the tail of the log; otherwise we would only end up seeing
+        # the beginning of it once the logging library later truncates it to the
+        # STACKDRIVER_LOG_MESSAGE_LIMIT.
+        logs.log_warn(qemu_log[-64 * 1024:])
```

Comment:
nit: put `64 * 1024` in a constant in a suitable platforms/fuchsia/ module and use in both places.

Reply:
Done. I've put it inside the `undercoat` module since once the migration to undercoat is complete the other files will all be removed.